### PR TITLE
Chore(repo): Introduce script for package graduation

### PR DESCRIPTION
**Pull request by literat**
* make stable version from prerelease version

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -62,9 +62,13 @@ node_modules: package.json ## install node modules
 ## --- 🚀 Release management ----------------------------------------------------
 
 .PHONY: prerelease
-prerelease: ## create a new pre-release
+prerelease: ## release current changes as prerelease versions
 	@bin/make/prerelease.sh
 
+.PHONY: graduate
+graduate: ## graduate prerelease versioned packages to stable versions
```
Comment (adamkudrna) [inline Makefile:69]: Does it mean only pre-release versions will be graduated and other packages will remain intact?

**Review by adamkudrna — COMMENTED** [review]
Looks good, though I'm not going to test it 🙂.

File: Makefile (hunk 1 above)
Comment (literat) [reply Makefile:69]: Yes, only pre-release versions.

**Comment by literat** [comment]
> Looks good, though I'm not going to test it 🙂.

I tested this manually.

**Pull request merged**